static/javascript/launch.js

```diff
@@ -366,7 +369,6 @@ async function loadSDSDatasetMetadata(survey_id, period_id) {
 
 function handleNoSupplementaryData() {
   showSupplementaryData(false);
-  showSubmitFlushButtons(false);
```

Comment:
Why was this removed? There is something odd happening now when I try to flush the data, the flush button opens a schema but it seems to be happening in staging as well so maybe just some bug.

Reply:
Had a quick discussion about this but just to confirm - I wanted to guard against `404` responses from the mock SDS service, as we're now calling that API **every** time for **any** schema now. However, it looks like `404`s are [gracefully handled by te backend](https://github.com/ONSdigital/eq-questionnaire-launcher/blob/6d17e39e6234fdcf282d741951d429a453977cc1/surveys/surveys.go#L261) and return an empty list with a `200` status code to the client.

Will add this back in so that any response from the mock SDS that's not a `200` or `404` will cause a failure and will therefore hide the buttons 👍